libcxx/include/__cxx03/CMakeLists.txt

```diff
@@ -1021,17 +1021,8 @@ set(files
 configure_file("__config_site.in" "${LIBCXX_GENERATED_INCLUDE_TARGET_DIR}/__config_site" @ONLY)
```

Comment:
@ldionne Unless I am missing something it appears this file is unused. If this is correct I'll open a new PR to delete it and will rebase on top of that.

Reply:
This isn't used _yet_.

Reply:
@philnik777 Should be be modifying the C++03 `CMakeLists.txt` at all?